src/models/userHelpers.js

```diff
@@ -638,43 +647,50 @@ async function getExistingUsers(d2, options = {}) {
     });
     return users;
 }
-function addItems(items1, items2, shouldAdd) {
+
+function addItems(items1, items2, shouldAdd, updateStrategy) {
     if (!shouldAdd) {
         return items1;
     } else {
-        return _(items1)
-            .unionBy(items2, element => element.id)
-            .value();
+        return updateStrategy === "merge"
+            ? _(items1)
```

Comment:
[very minor, no need to change] When only one lodash function if used, the non-wrapped version: `_.unionBy(...)` uses only one line. The pro of doing it this is is that it's very easy to add more lodash methods to the chain.